### PR TITLE
AC Automated Test - LRAC-8351 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8351

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileAudience.testcase [hunk 1]
```diff
@@ -30,12 +30,13 @@ definition {
...
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8351 | Test Summary: Page profile audience card shows known and anonymous individuals as well as segmented/unsegmented individuals"
-	@ignore = "true"
 	@priority = "5"
 	test ShowsKnownAnonymousIndividualsAsWellAsSegmentedUnsegmentedIndividuals {
+		task ("Go to the site and assert that Audience Card shows Known/Anonymous and Segmented/Unsegmented Individuals") {
+			ACUtils.launchAC();
 
-		// TODO LRAC-8351 ShowsKnownAnonymousIndividualsAsWellAsSegmentedUnsegmentedIndividuals
-
+			ACSites.assertAudienceCardItems(siteName = "Order History");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileAudience.testcase:38]: hey @AfonsoHSantos , where are you creating this site name "Order History"?

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+		AssertElementPresent(
+			key_legendName = "Segmented",
+			locator1 = "ACSites#AUDIENCE_LEGEND_ITEM");
+
+		AssertElementPresent(
+			key_legendName = "Unsegmented",
+			locator1 = "ACSites#AUDIENCE_LEGEND_ITEM");
+	}
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro:26]: Hi @AfonsoHSantos , this test case is not expected to validate the legends/strings available in the Audience card, we are expecting to see values related to Known/Anonymous Individuals, and Segmented/Unsegmented.


Please, take a look on this test case `AssetsBlogs#AudienceCardShowsExpectedAmountKnowAndAnonymousIndividualsInBlogs`, the logic will be very similar.

Thanks!

**Comment by alexcandidos** [comment]
hi @AfonsoHSantos , please take a look at my comments. Thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @alexcandidos, I made the requested changes. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileAudience.testcase [hunk 3]
```diff
@@ -30,12 +30,115 @@ definition {
...
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8351 | Test Summary: Page profile audience card shows known and anonymous individuals as well as segmented/unsegmented individuals"
-	@ignore = "true"
 	@priority = "5"
 	test ShowsKnownAnonymousIndividualsAsWellAsSegmentedUnsegmentedIndividuals {
+		task ("Add page and Blog for interaction with the AC") {
+			ACUtils.addSiteAndPage();
 
-		// TODO LRAC-8351 ShowsKnownAnonymousIndividualsAsWellAsSegmentedUnsegmentedIndividuals
+			ACUtils.createBlogsAndAddToPage();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileAudience.testcase:38]: hi @AfonsoHSantos , I don't think we need to create a blog on this test case... what do you think?

Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileAudience.testcase:38]: Hmm OK! I'll check that.

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
Rebased and resent via https://github.com/liferay-qa/liferay-portal/pull/788.

@AfonsoHSantos you need to fix the existing opened PRs since you updated/deleted your old repo.

**Pull request closed without merging**